### PR TITLE
Fix syscoumns test failure

### PR DESCRIPTION
**Pull request by ZhenyeLee**
Fix syscoumns test failure

Task:None
Signed-off-by: Zhenye Li <zhenye@amazon.com>

### Description

The helper function in prepare file searching object_name by oid in both pg_class and pg_proc. However, pg_class and pg_proc may have the same oid and it cause test failure. In this PR, we break down the search into to functions, one is in pg_class, and another one is in pg_proc. And also removed ``order by id`` from verification file, because oid can be changed after MVU.
 
### Issues Resolved

syscoumns test failure

### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by 2jungkook** [comment]
FYI, PG guarantees the uniqueness of OID only within a system relation (https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish/blob/BABEL_2_X_DEV__PG_14_X/src/backend/catalog/catalog.c#L306).

**Comment by 2jungkook** [comment]
Could you revert https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/598 in this PR so that we can see the issue is actually resolved?

**Review by 2jungkook — APPROVED** [review]

**Comment by 2jungkook** [comment]
Waiting for the original author's approval and an internal code review.

**Review by thephantomthief — APPROVED** [review]

**Pull request merged**
